roles/multi-tenancy/defaults/main.yml

```diff
@@ -13,13 +13,13 @@ V4MT_MODE: "schema"
 
 V4MT_TENANT_IDS: null
 
-# Optional job paramter
+# Optional job parameter
 V4MT_PROVIDER_PASSWORD: null
 
-V4MT_CAS_WORKER_COUNT: 0
-V4MT_CAS_BACKUP: 0
```

Comment:
Are these breaking anything be removing them? I know we'll be removing the provider based CAS soon, but having these values removed now seems a bit premature. Should they not stay? I don't see equivalent items in the code. Until the base CAS provider has been removed all items related to that code base that are relevant must stay.

Reply:
No nothing is breaking by removing this variables. These were moved over to be part of the new variable. Under V4MT_TENANT_CAS_CUSTOMIZATION see: V4MT_CFG_CAS_WORKER_COUNT, V4MT_CFG_CAS_ENABLE_BACKUP_CONTROLLER

Reply:
So the provider based CAS server does not use or have worker and or backups? Again, asking as this was there for a reason. So guessing that the provider does need these until CAS does not have this in play during multi-tenancy correct? Just covering the bases as folks might not employ multi-tenancy so these would be valid.

Reply:
These are not the provider variables. The provider side variables V4_CFG_CAS_WORKER_COUNT and V4_CFG_CAS_ENABLE_BACKUP_CONTROLLER are still in place and should not be affected. The variables I removed and moved were tenant specific starting with 'V4MT'.

Reply:
Got it. Sorry should have looked at the prefix more closely. You did identify these as V4MT items. Thx.